app/controllers/sessions_controller.rb

```diff
@@ -20,12 +20,12 @@ def create
         flash[:notice] = "The credentials you have entered are invalid"
         redirect_to "/login"
       end
-    end 
+    end
   end
 
   def destroy
     session.delete(:user_id)
-    flash[:notice] = "You have logged out"
+    flash[:notice] = "You are now logged out"
```

Comment:
Spending so much time working on and thinking about UX got me thinking about some of the language like this and I just tweaked it to make it seem more friendly.